en/index.md

```diff
@@ -14,62 +14,60 @@ This guide contains everything you need to assemble, configure, and safely fly a
 
 :::warning
 This guide is for the _development_ version of PX4 (`main` branch).
-Use the **Version** selector for the current _stable_ version.
+Use the **Version** selector to find the current _stable_ version.
 
 Documented changes since the stable release are captured in the evolving [release note](releases/main.md).
 :::
 
 ## How Do I Get Started?
 
-[Getting Started](getting_started/index.md) should be read by all users!
+[Basic Concepts](../getting_started/px4_basic_concepts.md) should be read by all users!
 It provides an overview of PX4, including features provided by the flight stack (flight modes and safety features) and the supported hardware (flight controller, vehicles, airframes, telemetry systems, RC control systems).

```

Comment:
Just caught my attention but vehicle vs airframe or rover airframe seem weird. We might want to consider renaming to limit confusion, just a thought.

Reply:
Yeah, this isn't right. I usually prefer to use vehicle type or vehicle frame instead of "airframe", in particular when talking about a non-flying vehicle..

```suggestion
It provides an overview of PX4, including features provided by the flight stack (flight modes and safety features) and the supported hardware (flight controller, vehicle types, telemetry systems, RC control systems).

```